### PR TITLE
Cleanup feature template to be real

### PR DESCRIPTION
**Pull request by timlevett**
Added in some notes to cleanup some gotchas

### CODE REVIEW

File: uw-frame-components/staticFeeds/features_template.json [hunk 1]
```diff
@@ -16,16 +16,14 @@
       "startMonth": 7,
       "startDay": 11,
       "endYear": 2015,
-      "endMonth": 7,
+      "endMonth": 7, (0 based index)
```
Comment (apetro) [inline uw-frame-components/staticFeeds/features_template.json]: How safe is this approach to adding comments to `JSON`? Googling about this [feels bleak](http://stackoverflow.com/questions/244777/can-i-use-comments-inside-a-json-file)?

Comment (jhanstra) [inline uw-frame-components/staticFeeds/features_template.json]: Agreed, that seems sketchy

Comment (timlevett) [inline uw-frame-components/staticFeeds/features_template.json]: This is a template for `features.json`, not the actual JSON. ;)

Comment (apetro) [inline uw-frame-components/staticFeeds/features_template.json]: Ah, hadn't caught that it was a template.

:+1: 

Moving this into the documentation sounds like a good idea, but I don't think it needs to block this bit of inline documentation.

**Pull request merged**